cmd/upgrade.go

```diff
@@ -10,7 +10,6 @@ import (
 )
 
 func init() {
-	upgrade.Flags().StringP("config", "c", "fogg.json", "Use this to override the fogg config file.")
```

Comment:
Even when we are yaml-only we should still allow people to specify a config file.